# R Square - Score - 
- tells us what percentage of the variation in target variable is explained by model.
- higher the better

In [13]:
from sklearn.metrics import r2_score, mean_squared_error, mean_absolute_error

import numpy as np

actual = [10, 15, 18]
predicted = [12, 14, 16]
print('R Squared:' , r2_score(actual,predicted))
print('RMSE:',np.sqrt(mean_squared_error(actual, predicted)))
print('MAE:', mean_absolute_error(actual, predicted))

R Squared: 0.7244897959183674
RMSE: 1.7320508075688772
MAE: 1.6666666666666667


i.e the model explains 72% of the variance in the model.